src/components/DatepickerRange/index.stories.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+import fr from 'date-fns/locale/fr';
+import DatepickerRange from '.';
+import Heading from '../Heading';
+import { Code } from '../../storybook-utils';
+
+const stories = storiesOf('DatepickerRange', module);
+
+const dateKnob = (name, defaultValue) => {
+  const date = text(name, defaultValue);
+  return new Date(date);
+};
+
+stories.add('All states', () => (
+  <>
+    <Heading>DatepickerRange</Heading>
+
+    <Heading level={2}>With value</Heading>
+    <DatepickerRange
+      id='default'
+      value={new Date('2020-08-04')}
+      isClearable
+      startPlaceholder='Start date range'
+      endPlaceholder='End date range'
+    />
+    <Code>{`
+    <DatepickerDay id='default' value={new Date('2020-08-04')} isClearable startPlaceholder='Start date range' endPlaceholder='End date range'/>
+    `}</Code>
+
+    <Heading level={2}>Pink version</Heading>
+    <DatepickerRange
+      id='default'
+      value={new Date('2020-08-04')}
+      isClearable
+      startPlaceholder='Start date range'
+      endPlaceholder='End date range'
+      isPink
+    />
+    <Code>{`
+    <DatepickerDay id='default' value={new Date('2020-08-04')} isClearable startPlaceholder='Start date range' endPlaceholder='End date range' isBlue/>
```

Comment:
```suggestion
    <DatepickerDay id='default' value={new Date('2020-08-04')} isClearable startPlaceholder='Start date range' endPlaceholder='End date range' isPink/>
```

Reply:
Would it not be better to use brand= 'primary` or `secondary` so that in the future if the colors change we can keep the same naming?

Reply:
As I said on Friday, I would even go for something wider. Let's pretend it's in TypeScript already, we could have a prop named `accentColor`:
```typescript
{
  accentColor: 
    | { brand: 'primary' | 'secondary' | 'tertiary' }
    | { color: string }
}
```
For the branded variant, we could even go further and allow most of the variants we have [here](https://flamingo.pages.dev/?path=/story/design-system-colors--interface-colors) and [here](https://flamingo.pages.dev/?path=/story/design-system-colors--brand-colors). For instance things like `passenger-primary`, `driver-primary`, `passenger-secondary`, `driver-secondary`, or `brand-red-3`.

Reply:
But this is maybe a little ambitious for now and we could wait for a bigger refactor to implement something like that 😅

Reply:
Yes I guess it will be best to have a separate PR for that